requirements/common.in

```diff
@@ -34,9 +34,13 @@ backports-abc==0.5
 # Needed for Tornado 4 compatibility
 backports.ssl-match-hostname==3.5.0.1
 
-# Needed for S3 file uploads
+# Keep boto since puppet needs boto.utils.get_instance_metadata and this is not implemented in boto3
+# Reference: https://github.com/boto/boto3/issues/313
```

Comment:
This can be removed after https://github.com/zulip/zulip/pull/10972 is merged, and so there is no puppet on dev environment. For prod environment, the boto library should be added declaratively via puppet instead.